runtime/src/commonMain/kotlin/pbandk/internal/json/JsonValueMarshaller.kt

```diff
@@ -105,4 +106,23 @@ internal class JsonValueMarshaller(private val jsonConfig: JsonConfig) {
             writeValue(k, keyType).content to writeValue(v, valueType)
         }
     }
+
+    private fun Value.toElement(): JsonElement = when (this.kind) {
+        is Value.Kind.StringValue -> writeString(this.stringValue!!)
```

Comment:
🔧 You can use `this.kind.value` instead of `this.stringValue!!` to avoid having to use the `!!`.